libnavigation-core/api/current.txt

```diff
@@ -130,10 +130,11 @@ package com.mapbox.navigation.core.history {
   }
 
   public final class MapboxHistoryRecorder {
-    method public String? fileDirectory();
+    method public String getFileDirectory();
```

Comment:
Can this stay null?

When it is working as expected, it should be non-null. When it is not assigned it is null.